bot/exts/evergreen/recommend_game.py

```diff
@@ -33,18 +34,18 @@ async def recommend_game(self, ctx: commands.Context) -> None:
         game = game_recs[self.index]
         self.index += 1
 
-        author = self.bot.get_user(int(game['author']))
+        author = self.bot.get_user(int(game["author"]))
 
         # Creating and formatting Embed
         embed = discord.Embed(color=discord.Colour.blue())
         if author is not None:
             embed.set_author(name=author.name, icon_url=author.avatar_url)
-        embed.set_image(url=game['image'])
-        embed.add_field(name='Recommendation: ' + game['title'] + '\n' + game['link'], value=game['description'])
+        embed.set_image(url=game["image"])
+        embed.add_field(name="Recommendation: " + game["title"] + "\n" + game["link"], value=game["description"])
```

Comment:
Name can be an f-string.
```suggestion
        embed.add_field(name=f"Recommendation: {game['title']}\n{game['link']}", value=game["description"])
```